tools/themis/src/utils.rs

```diff
@@ -22,12 +22,17 @@ pub fn parse_workspace() -> anyhow::Result<Workspace> {
         .cloned()
         .filter(|package| metadata.workspace_members.contains(&package.id))
         .map(|package| {
-            let raw = read_toml(&package.manifest_path)?;
+            let raw = match read_toml(&package.manifest_path)? {
+                Some(raw) => raw,
+                _ => return Err(anyhow::anyhow!("failed to read package manifest")),
```

Comment:
This is a bit unintuitive.  I can get some failures from line 25 and another failure here.  I wonder if we need to use stronger return types in `read_toml()`.

Reply:
At this point, line 25 returns all errors but ENOENT. Just needed to make this error report was more distinctive. Should be now.